pandas/core/tools/timedeltas.py

```diff
@@ -64,6 +69,31 @@ def to_timedelta(arg, unit='ns', box=True, errors='raise'):
     TimedeltaIndex(['0 days', '1 days', '2 days', '3 days', '4 days'],
                    dtype='timedelta64[ns]', freq=None)
 
+    For `M` and `Y` units, `1M = 30D` and 1Y = 365D:
```

Comment:
So there's an issue to deprecate `'M'` and `'Y'` for Timedelta, so I don't think it's a good idea to advertise these argument that are slated for deprecation. 

https://github.com/pandas-dev/pandas/issues/16344

Reply:
@mroeschke  sorry wasn't aware about the issue - ill remove it. Im thinking giving example for all the other units would be overkill - maybe a couple more ? 
Maybe a reference to DateOffset in the description or examples would help ?

Reply:
If you're up for it, you could address #16344 first and then add a note here saying "Day frequencies and higher are only supported.

Reply:
Just sent an initial PR for #16344 , still have work to do as its my first time fixing deprecation issues before